functions.php

```diff
@@ -45,6 +45,21 @@ function escapade_add_mobile_menu() {
 add_action( 'primer_header', 'escapade_add_mobile_menu', 0 );
 
 /**
+ * Enqueue Escapade scripts
+ *
+ * @action wp_enqueue_scripts
+ * @since NEXT
+ */
+function escapade_load_scripts() {
+
+	$suffix = SCRIPT_DEBUG ? '' : '.min';
+
+	wp_enqueue_script( 'sidebar.js', get_stylesheet_directory_uri() . "/assets/js/sidebar$suffix.js", [ 'jquery' ], 'all', true );
```

Comment:
@EvanHerman A few issues with this line:

1. The handle should be theme-specific to prevent collisions: `escapade-sidebar`
2. Curly braces help readability by showing where the expanding var name ends: `sidebar{$suffix}.js`
3. Unlike `wp_enqueue_style()` the 4th param in `wp_enqueue_script()` is for a version: `PRIMER_CHILD_VERSION`